VirtoCommerce.CatalogModule.Web/Scripts/blades/images-add.js

```diff
@@ -39,7 +40,7 @@
                         removeAfterUpload: true
                     });
 
-                    uploader.url = getImageUrl(item.code, blade.imageType).relative;
+                    uploader.url = getImageUrl(item.code, blade.imageType, item.category).relative;
```

Comment:
This controller reused in other places. and `item` here might be a category type.
I this is a not correct way to try get the storage path from element here, the better solution would be to pass a storage folder path as a special parameter to this controller.